### ISSUE TITLE
ValueError with --scan-hosts, easy fix

### ISSUE THREAD

**Issue by jayekub** [postlund/pyatv#405]
just trying out pyatv/apple_tv_mrp for the first time. a regular scan wasn't working so i tried the -s flag and got this:

```
$ atvremote scan -s 192.168.0.6
ERROR: Exception in callback _SelectorDatagramTransport._read_ready()
handle: <Handle _SelectorDatagramTransport._read_ready()>
Traceback (most recent call last):
  File "/usr/lib/python3.6/asyncio/events.py", line 145, in _run
    self._callback(*self._args)
  File "/usr/lib/python3.6/asyncio/selector_events.py", line 1068, in _read_ready
    self._protocol.datagram_received(data, addr)
  File "/home/jacob/.local/lib/python3.6/site-packages/pyatv/udns.py", line 229, in datagram_received
    self.result = DnsMessage().unpack(data)
  File "/home/jacob/.local/lib/python3.6/site-packages/pyatv/udns.py", line 147, in unpack
    ptr, data = unpack_rr(ptr, msg)
  File "/home/jacob/.local/lib/python3.6/site-packages/pyatv/udns.py", line 98, in unpack_rr
    rd_data = parse_txt_dict(rd_data, msg)
  File "/home/jacob/.local/lib/python3.6/site-packages/pyatv/udns.py", line 58, in parse_txt_dict
    key, value = prop.split(b'=')
ValueError: too many values to unpack (expected 2)
Scan Results
========================================
```

i added a print(str(prop)) above the failing line and found it was having trouble with this:

b'BluetoothAddress=\xc8\xd0\x83\xca=~'

changing the line in question like so fixed the issue:

        key, value = prop.split(b'=', 1)

don't have a good setup for making a pull request at the moment, hopefully you don't mind taking the fix this way :)

**Comment by postlund** [comment]
Ah, yes. This is clearly a bug. I will add or update an existing test to cover and fix it. Thanks for reporting! 👍

**Comment by postlund** [comment]
Submitted a PR to fix this: #407
